test/helpers/name.go

```diff
@@ -17,14 +17,24 @@ limitations under the License.
 package helpers
 
 import (
+	"math/rand"
```

Comment:
I think it's recommended to use `https://golang.org/pkg/crypto/rand/`

Reply:
I agree with it. But as per https://yourbasic.org/golang/crypto-rand-int/, since `crypto/rand` does not have `Seed` method, it's a bit verbose if we want to use it. So I still prefer `math/rand` here.

Reply:
They can be combined, but also for reasons we're doing here (generating random service names for tests) regular rand should suffice.

Reply:
sgtm